creator-node/src/snapbackSM/peerSetManager.js

```diff
@@ -193,14 +203,58 @@ class PeerSetManager {
    * @param {string} endpoint
    * @returns {Object} the /health_check/verbose response
    */
-  async determinePeerHealth (endpoint) {
+  async getPeerHealth (endpoint) {
     // Axios request will throw on timeout or non-200 response
-    return axios({
+    const resp = await axios({
       baseURL: endpoint,
       url: '/health_check/verbose',
       method: 'get',
       timeout: PEER_HEALTH_CHECK_REQUEST_TIMEOUT
     })
+
+    return resp.data.data
+  }
+
+  /**
+   * Takes data off the verbose health check response and determines the peer heatlh
+   * @param {Object} verboseHealthCheckResp verbose health check response
+   *
+   * TODO: consolidate CreatorNodeSelection + peer set health check calculation logic
+   */
+  determinePeerHealth (verboseHealthCheckResp) {
```

Comment:
what happens if any of these required fields are missing? seems reasonable to mark as unhealthy to me? currently your code will treat it as healthy. wdyt

Reply:
discussed offline - if the values are not present for some reason, would not like to preemptively mark node as unhealthy as this may be a false negative. leaving as is